Sources/MenuAnimator.swift

```diff
@@ -179,7 +191,10 @@ class MenuInteractiveTransition: NSObject, UIViewControllerInteractiveTransition
     //MARK: - Private methods
     //
     private func createSnapshotView(from: UIView) -> UIView {
-        let snapshotView = from.snapshotView(afterScreenUpdates: true)!
+        let snapshotViewOptional = from.snapshotView(afterScreenUpdates: true)
+        guard let snapshotView = snapshotViewOptional else {
```

Comment:
Let's write here 
`guard let snapshotView = from.snapshotView(afterScreenUpdates: true) else` and remove `snapshotViewOptional`